test/run-tests.js

```diff
@@ -131,7 +131,7 @@ function run_compress_tests() {
                 input.figure_out_scope(test.mangle);
                 input.expand_names(test.mangle);
             }
-            var cmp = new U.Compressor(options, true);
+            var cmp = new U.Compressor(options, options.defaults === undefined ? true : !options.defaults);
```

Comment:
(Sorry for the delayed response - was out for badminton.)

As mentioned in https://github.com/mishoo/UglifyJS2/pull/2820#issuecomment-359150504, can we eliminate this?

Reply:
That's something I'd like in the test framework to simplify some new tests. By specifying `defaults: true` one can create a test case with default minify compress options.

https://github.com/mishoo/UglifyJS2/pull/2820/files#diff-e8e8f44d374c70df96a0dfa780b1447dR50